From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
From: Buildroot <buildroot@buildroot.org>
Date: Mon, 28 Sep 2026 00:00:00 +0000
Subject: [PATCH] cmake: update minimum required version to 3.5

CMake 3.x removed support for CMake < 3.5. Update the minimum
required version to avoid configuration errors.

Signed-off-by: Buildroot <buildroot@buildroot.org>
---
 CMakeLists.txt | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1111111..2222222 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -1,7 +1,7 @@
 # SoX Resampler Library       Copyright (c) 2007-13 robs@users.sourceforge.net
 # Licence for this file: LGPL v2.1                  See LICENCE for details.

-cmake_minimum_required (VERSION 2.8 FATAL_ERROR)
+cmake_minimum_required (VERSION 3.5 FATAL_ERROR)

 project (soxr C)
 set (DESCRIPTION_SUMMARY "High quality, one-dimensional sample-rate conversion library")
--
2.20.1
